tests/integration/test_drop_replica_with_auxiliary_zookeepers/test.py

```diff
@@ -0,0 +1,70 @@
+import time
+
+import helpers.client as client
+import pytest
+from helpers.cluster import ClickHouseCluster
+from helpers.client import QueryRuntimeException
+from helpers.test_tools import TSV
+
+cluster = ClickHouseCluster(__file__)
+node1 = cluster.add_instance(
+    "node1",
+    main_configs=["configs/zookeeper_config.xml", "configs/remote_servers.xml"],
+    with_zookeeper=True,
+    use_keeper=False,
+    stay_alive=True,
+)
+node2 = cluster.add_instance(
+    "node2",
+    main_configs=["configs/zookeeper_config.xml", "configs/remote_servers.xml"],
+    with_zookeeper=True,
+    use_keeper=False,
+    stay_alive=True,
+)
+
+
+@pytest.fixture(scope="module")
+def started_cluster():
+    try:
+        cluster.start()
+
+        yield cluster
+
+    except Exception as ex:
+        print(ex)
+
+    finally:
+        cluster.shutdown()
+
+
+def drop_table(nodes, table_name):
+    for node in nodes:
+        node.query("DROP TABLE IF EXISTS {} NO DELAY".format(table_name))
+
+
+def test_drop_replica_in_auxiliary_zookeeper(started_cluster):
+    drop_table([node1, node2], "test_auxiliary_zookeeper")
+    for node in [node1, node2]:
+        node.query(
+            """
+                CREATE TABLE test_auxiliary_zookeeper(a Int32)
+                ENGINE = ReplicatedMergeTree('zookeeper2:/clickhouse/tables/test/test_auxiliary_zookeeper', '{replica}')
+                ORDER BY a;
+            """.format(
+                replica=node.name
+            )
+        )
+
+    # stop node2 server
+    node2.stop_clickhouse()
+    time.sleep(5)
```

Comment:
It's not reliable, the test will be flaky. But it was supposed to avoid `Can't drop replica because it's active`, right? It can be done reliably by checking result of `select ... from system.replicas ...` or `select ... from system.zookeeper ...` in a loop

Reply:
ok.

Reply:
https://github.com/ClickHouse/ClickHouse/blob/b0da0cffe75a6570288ed5180ca9f9b68168f6d6/tests/integration/test_drop_replica_with_auxiliary_zookeepers/test.py#L62